felix/bpf-gpl/tc.c

```diff
@@ -1191,31 +1192,12 @@ int calico_tc_skb_send_icmp_replies(struct __sk_buff *skb)
 	return TC_ACT_SHOT;
 }
 
-SEC("classifier/tc/prologue_v6")
-int calico_tc_v6(struct __sk_buff *skb)
+SEC("classifier/tc/drop")
+int calico_tc_skb_drop(struct __sk_buff *skb)
 {
-	CALI_DEBUG("Entering IPv6 prologue program");
-	// TODO: Replace this logic with the proper implementation, and finally a tail call
-	// to the policy program
-	return TC_ACT_UNSPEC;
-}
-
-SEC("classifier/tc/accept_v6")
-int calico_tc_v6_skb_accepted_entrypoint(struct __sk_buff *skb)
-{
-	CALI_DEBUG("Entering IPv6 accepted program");
-	// TODO: Implement the logic for accepted packets by the policy program
-	// We should not reach here since no tail call happens to this program
-	return TC_ACT_UNSPEC;
-}
-
-SEC("classifier/tc/icmp_v6")
-int calico_tc_v6_skb_send_icmp_replies(struct __sk_buff *skb)
-{
-	CALI_DEBUG("Entering IPv6 icmp program");
-	// TODO: Implement the logic for accepted icmp packets by the policy program
+	CALI_DEBUG("Entering calico_tc_skb_drop\n");
 	// We should not reach here since no tail call happens to this program
```

Comment:
Out of date comment?

Reply:
I think this is a valid comment for OS since we are not really using the drop program. Right?